docs/installation.rst

```diff
@@ -336,6 +334,22 @@ the path to the source directory.
                            if the build is on a separate filesystem from the source, as
                            required on some HPC systems.
 
+- Legacy CUDA version of QMCPACK
+
+  ::
+
+    QMC_CUDA      Enable legacy CUDA code path for NVIDIA GPU acceleration (default: OFF)
+    QMC_CUDA2HIP  AMD GPU support by translating CUDA to HIP (default: OFF)
+
+- Auxiliary Field Quantum Monte Carlo
+
+  ::
+
+    BUILD_AFQMC  Build AFQMC-capable code (default: OFF)
+    ENABLE_HIP   AMD GPU support through HIP (default: OFF)
+    ENABLE_CUDA  NVIDIA GPU support through CUDA (default: OFF)
```

Comment:
Although ENABLE_HIP is AFQMC exclusive, we should present as a GPU option not a AFQMC option
ENABLE_CUDA is not AFQMC only.